equinix/resource_fabric_connection.go

```diff
@@ -18,6 +19,574 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func FabricConnectionResourceSchema() map[string]*schema.Schema {
```

Comment:
This was previously a private in the package: `createFabricConnectionResourceSchema`. This is preferable since we don't want or expect outside users to depend on this function. Removing exported functions at a later date is a breaking change.

In the future migration to `internal/resources/fabric/connection/` (perhaps `internal/services/fabric/connection/`) the function could be exported and shared between other packages in the Terraform provider, without including the function in the public signature of the `github.com/equinix/terraform-provider-equinix/equinix` package.  That package will minimally expose the `Provider`.

Reply:
> Removing exported functions at a later date is a breaking change

because this project was released at 1.0.0 in the past.

https://0ver.org/ 🙂

Reply:
Copy that. Heard and will fix here and take that review forward into further Provider changes.